src/routes/backoffice.ts

```diff
@@ -361,6 +361,8 @@ export const processQueuedBooking = async (
         sender_bic: booking.recipient_bic,
         recipient_bic: booking.sender_bic,
         id: booking.id.split("-").reverse().join("-"),
+        transaction_id: null,
+        return_transaction_id: booking.transaction_id,
```

Comment:
❓ Why do we need this?

Reply:
it's needed to match DDR with corresponding DD:
`dd. transaction_id === ddr. return_transaction_id`